FileReeder.java

```diff
@@ -0,0 +1,33 @@
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+
+public class FileReeder {
```

Comment:
Perhaps add some documentation to the file, like JavaDoc, so it's clear for others how they should use the class. Which input to use, what is the result?